src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -195,9 +195,16 @@ void JwtAuthenticator::VerifyKey(const PubkeyCacheItem& issuer_item) {
     return;
   }
 
-  // TODO: check forward_payload_header.
   headers_->addReferenceKey(kJwtPayloadKey, jwt_->PayloadStrBase64Url());
```

Comment:
Please add comment why this payload key is needed and link to this issue (deprecate EUC policy)

https://github.com/istio/istio/issues/4744

Reply:
Done: https://github.com/istio/proxy/pull/1364/commits/e83418b1ce89f450d55a8a685ab5524f165fdf2a.

Reply:
The comment is added in the https://github.com/istio/proxy/pull/1364.